### DISCUSSION TITLE
ww3_multi run error

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#933] [Q&A] [answered]
When i run ww3_ multi (ww3 v6.07) i get the following message error:

![Figure1](https://user-images.githubusercontent.com/78570105/222988049-f77a9a5e-3c55-4afe-95b8-da6e6eb29388.png)

This is the content of the www_multi.inp file that i am using:

![Figure2](https://user-images.githubusercontent.com/78570105/222988201-6b3dd54a-ae49-4fc1-b1a4-de18d2ece336.png)

My input data is: mod_def.Global, mod_def.AtlSur, restart.Global and restart.AtlSur. Any idea what could be going on?

Thank you in advance.

**Comment by EmilioEchevarria** [comment, marked as answer]
Hi,

Have a look at the example .inp files provided. You say you're using WW3 v6.07, so this is an example of ww3_multi.inp file: https://github.com/NOAA-EMC/WW3/blob/6.07/model/inp/ww3_multi.inp. You can also have a look at the [regtests](https://github.com/NOAA-EMC/WW3/tree/6.07/regtests) to see examples of how to use ww3_multi. 

In your case, remove the line `'AtlSur' 1`.  Also, do you need track output, boundary output, etc? Consider setting the time step for those to `0`.

Hope that helps, 
Emilio
